src/module/actor/lancer-actor.ts

```diff
@@ -838,17 +857,17 @@ export class LancerActor<T extends LancerActorType> extends Actor {
       this._prev_derived = dr;
     } else {
       // Otherwise, grab existing/prior
-      dr = this._prev_derived
+      dr = this._prev_derived;
     }
     this.data.data.derived = dr;
 
-    // Update our known values now, synchronously. 
-    dr.current_hp.value = this.data.data.current_hp
-    if(this.data.type != EntryType.PILOT) {
-      let md = this.data.data as RegEntryTypes<EntryType.MECH | EntryType.NPC | EntryType.DEPLOYABLE>;
+    // Update our known values now, synchronously.
+    dr.current_hp.value = this.data.data.current_hp;
+    if (this.data.type != EntryType.PILOT) {
```

Comment:
`if (!this.is_pilot())`, perhaps? there's a few more of these

Reply:
Looks like I missed that on. I ran a ripgrep for the pattern, but I'll get that fixed.

Reply:
I believe I got them all.